DNN Platform/Admin Modules/Dnn.Modules.Console/Components/ConsoleController.cs

```diff
@@ -47,10 +47,12 @@ public class ConsoleController
         /// <returns>A list with different icon types</returns>
         public static IList<string> GetSizeValues()
         {
-            IList<string> returnValue = new List<string>();
-            returnValue.Add("IconFile");
-            returnValue.Add("IconFileLarge");
-            returnValue.Add("IconNone");
+            IList<string> returnValue = new List<string>
+            {
+                "IconFile",
+                "IconFileLarge",
+                "IconNone"
+            };
```

Comment:
I don't have a preference in the style, but I see a lot of changes like this. I am more curious if these were recommendations from tools such as Resharper that made these recommendations. Should we start discussing a style guide that we want to start enforcing?